src/common/options.cc

```diff
@@ -459,6 +459,10 @@ std::vector<Option> get_global_options() {
 			  "whatever options or values are required for us to "
 			  "work will be passed as arguments."),
 
+    Option("max_striper_locks_per_obj", Option::TYPE_UINT, Option::LEVEL_BASIC)
```

Comment:
this should be LEVEL_DEV - a regular user shouldn't need to change it, just developers using libradosstriper

Reply:
fixed